deps/build.jl

```diff
@@ -5,94 +5,77 @@
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
-include("ipython.jl")
-const ipython, ipyvers = find_ipython()
+juliaprofiles = Array(String,0)
 
-if ipyvers < v"1.0.0-dev"
-    error("IPython 1.0 or later is required for IJulia, got $ipyvers instead")
-else
-    eprintln("Found IPython version $ipyvers ... ok.")
-end
+if @windows? true : false
+    using BinDeps
+    downloadsdir = "downloads"
+    pyinstalldir = normpath(pwd(),"usr","python34")
 
-#######################################################################
-# Create Julia profile for IPython and fix the config options.
-
-# create julia profile (no-op if we already have one)
-eprintln("Creating julia profile in IPython...")
-run(`$ipython profile create julia`)
-
-juliaprof = chomp(readall(`$ipython locate profile julia`))
-
-# set c.$s in prof file to val, or nothing if it is already set
-# unless overwrite is true
-function add_config(prof::String, s::String, val, overwrite=false)
-    p = joinpath(juliaprof, prof)
-    r = Regex(string("^[ \\t]*c\\.", replace(s, r"\.", "\\."), "\\s*=.*\$"), "m")
-    if isfile(p)
-        c = readall(p)
-        if ismatch(r, c)
-            m = replace(match(r, c).match, r"\s*$", "")
-            if !overwrite || m[search(m,'c'):end] == "c.$s = $val"
-                eprintln("(Existing $s setting in $prof is untouched.)")
-            else
-                eprintln("Changing $s to $val in $prof...")
-                open(p, "w") do f
-                    print(f, replace(c, r, old -> "# $old"))
-                    print(f, """
-c.$s = $val
-""")
-                end
-            end
-        else
-            eprintln("Adding $s = $val to $prof...")
-            open(p, "a") do f
-                print(f, """
-
-c.$s = $val
-""")
-            end
-        end
-    else
-        eprintln("Creating $prof with $s = $val...")
-        open(p, "w") do f
-            print(f, """
-c = get_config()
-c.$s = $val
-""")
-        end
+    if ispath(downloadsdir)
+        run(`cmd /C RD "$(normpath(pwd(),downloadsdir))" /S /Q`)
     end
-end
+    mkdir(downloadsdir)
 
-# add Julia kernel manager if we don't have one yet
-if VERSION >= v"0.3-"
-    binary_name = "julia"
-else
-    binary_name = "julia-basic"
+    pythonzipfilename = normpath(downloadsdir, "python-3.4.2.zip")
+    getpipfilename = normpath(downloadsdir,"get-pip.py")
+    run(download_cmd("https://sourceforge.net/projects/minimalportablepython/files/python-3.4.2.zip", "$pythonzipfilename"))
+    download("https://bootstrap.pypa.io/get-pip.py", "$getpipfilename")
```

Comment:
do we need this separately? don't the latest versions of python have ensurepip included?